deploy/jjb.sh

```diff
@@ -11,26 +11,73 @@
 #
 
 # error out in case a command fails
-set -e
+export PS4='\e[32m+ ${FUNCNAME:-main}@${BASH_SOURCE}:${LINENO} \e[0m'
```

Comment:
This is (currently) not a Bash script, only POSIX shell. So the above line needs adjusting, as well as the `function` usage.